keyboards/acheron/austin/ws2812.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "hal.h"
+#include "color.h"
+
+void set_leds_color_rgb(LED_TYPE color);
+void set_led_color_rgb(LED_TYPE color, int pos);
+void leds_init(void);
+
+
+ // This is what users will use to interface with this
+void ws2812_setleds(LED_TYPE *ledarray, uint16_t number_of_leds);
+void ws2812_setleds_rgbw(LED_TYPE *ledarray, uint16_t number_of_leds);
```

Comment:
This function no longer exists and can be safely removed.

Reply:
Change commited